packages/remix/scripts/injectDebugId.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-console */
+const { execSync } = require('child_process');
+
+function injectDebugId(buildPath) {
+  try {
+    execSync(`node ./node_modules/@sentry/cli/bin/sentry-cli sourcemaps inject ${buildPath}`);
```

Comment:
I can imagine some environments (e.g. monorepos) where this path lookup will not succeed.

The `@sentry/cli` package exports a function called `getPath` that we can use to get the binary path. Wdyt about using it here?

Additionally, I wouldn't call the node executable but the binary directly.

Reply:
Makes sense, updated 👍